src/shipping_queue/tables/ShippingHistoryTable.jsx

```diff
@@ -11,6 +11,7 @@ import { API } from "../../services/server";
 import { getSortFromModel } from "../utils/sortModelFunctions";
 import { PackShipProgress } from "../../common/CircularProgress";
 import { snackbarVariants, usePackShipSnackbar } from "../../common/Snackbar";
+import pdfMake from "pdfmake/build/pdfmake";
```

Comment:
I think pdfmake will need to be added to the package.json and pushed.

Reply:
Nvm, I see that it is in the package.json - it must have been added a while ago.